lapack.go

```diff
@@ -11,22 +11,57 @@ const None = 'N'
 type Job byte
```

Comment:
CompSV can though. It is pretty consistently (half a dozen checked) mean compute singular values.

Job could have its constants deleted since they are pretty broad - and the meaning of it seems to be "What job should I do?"